### DISCUSSION TITLE
Ctb touch screen new controls

### DISCUSSION THREAD

**Discussion by megadrive7** [ppy/osu#22260] [Ideas] [answered]
I personally think that the ctb gamemode would be much more playable if it behaved so that when you tap one side of the screen while already pressing the other, the catcher would pivot instead of staying in place, as it is currently. To pivot again you would just lift and then retap your finger. The current scheme is an improvement but still feels slightly unresponsive for me.

**Comment by bdach** [comment, marked as answer]
To clarify: you're saying that latest touch input should take priority, correct?

This is likely the way it'll be eventually implemented, just want to confirm understanding.

**Comment by peppy** [comment]
I'm not sure about this one. Would need more opinions from players.
